code/game/objects/items/toys.dm

```diff
@@ -40,21 +41,24 @@
 	..()
 	create_reagents(10)
 
-/obj/item/toy/balloon/attack__legacy__attackchain(mob/living/carbon/human/M as mob, mob/user as mob)
-	return
+/obj/item/toy/balloon/pre_attack(atom/target, mob/living/user, params)
+	if(..())
+		return FINISH_ATTACK
+	return FINISH_ATTACK
```

Comment:
```suggestion
/obj/item/toy/balloon/pre_attack(atom/target, mob/living/user, params)
	return FINISH_ATTACK
```
This is redundant here, since you are going to return the same thing either way.

Reply:
This has been done in a slightly different way.